Assets/Src/AuthController/JwtManagement/Converters/Google/GoogleJwtConverter.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using Src.AuthController.JwtManagement.Tokens;
+using Src.AuthController.REST.REST_Response_DTOs.Firebase.Google;
+using UnityEngine;
+
+namespace Src.AuthController.JwtManagement.Converters.Google
+{
+    public sealed class GoogleJwtConverter : IGoogleJwtConverter
+    {
+        public GoogleJwtStore FromGoogleAuthResponse(GoogleIdTokenResponse response)
+        {
+            var idToken = new JwtToken(response.IDToken,
+                Int32.MaxValue,
```

Comment:
PP: Instead of putting Int32.MaxValue here it would be better to create some logic for determining wheather or not token can expire. Anyway, current implementation will work. For nearest 70 years at least.